gh-pages/content/specification/3-kernel-api.md

```diff
@@ -471,10 +471,18 @@ such values.
 
 #### Asynchronous method invocation
 
+!!! bug "Largely un-tested"
+    Asynchronous operations are only partially supported in the various target languages, and is currently not widely
+    used. As such, support is not as "battle-tested" as the rest of the jsii interoperability features, and customers
+    may run into usability issues, unexpected bugs, or surprising behaviors when using async.
+
+    In particular, outstanding `Promise`s may not be able to make progress as expected due to specific implementation
+    details of the `@jsii/runtime` event loop, which can result in deadlock situations.
```

Comment:
NB: These are special "admonitions" on the particular MD renderer used by mkdocs here... Content needs to be indented by 4 spaces, and the first line is actually a magic token to declare the specific admonition style to be used.